app/uk/gov/hmrc/agentauthorisation/connectors/DesConnector.scala

```diff
@@ -39,13 +39,13 @@ object MtdItIdBusinessDetails {
 }
 
 @Singleton
-class DesConnector @Inject() (
+class DesConnector @Inject()(
   @Named("des-baseUrl") baseUrl: URL,
   @Named("des.authorizationToken") authorizationToken: String,
   @Named("des.environment") environment: String,
   http: HttpPost with HttpGet with HttpPut,
```

Comment:
if you use `DefaultHttpClient` that gives you all http methods and we can simply inject `httpClient: HttpClient`